package/etc/conf.d/conflib/syslog/app-tenable_syslog.conf

```diff
@@ -0,0 +1,18 @@
+block parser tenable_syslog-parser() {    
+ channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index('netfw')
+                sourcetype('tenable:nnm:vuln')
+                vendor_product("Tenable.nnm")
+            );              
+        };          
+
+   };
+};
+application tenable_syslog[sc4s-syslog] {
+    filter { 
+        program('nnm' type(string) flags(prefix))
```

Comment:
better syntax for eq is 
"$PROGRAM" eq "nnm"
we don't want nnmonitor for example if that was ever a thing